addons/_deps/sniffer-poller.js

```diff
@@ -6,11 +6,12 @@ const STATE_SONG_PLAYING = 4;
 const STATE_SONG_ENDING = 5;
 
 class SnifferPoller {
+	//Create variables and containers for poller data.
 	constructor(options = {}) {
 		var defaultOptions = {
 			ip: "127.0.0.1",
 			port: "9938",
-			interval: 900,
+			interval: 100,
```

Comment:
This interval is very fast, I don't think the addon service can keep up with it consistently.
It might be better to poll in onData instead of using a timer, just need to make sure to wait some time before polling when there was an error (sniffer not running for example).
The interval setting could be changed to be a delay before polling again.

Reply:
I have set this back to 900 for the standard, default release version. However I am convinced faster pollingrates still have benefits. Perhaps we should add it to the config-ui.js for easier editing? It gets referred to in script.js as well.